docs/README.md

```diff
@@ -47,13 +47,13 @@ For examples of how we run these services check out the [Nautilus](https://githu
 To run `lorre` and `conseil` with `sbt`, use the following:
 
 ```
-env SBT_OPTS="-Dconfig.file=conseil.conf" && sbt "runConseil"
-env SBT_OPTS="-Dconfig.file=conseil.conf" && sbt "runLorre alphanet"
+env SBT_OPTS="-Dconfig.file=conseil.conf" && sbt -J-Xss32m "runConseil"
```

Comment:
Have you checked how does it really work? We need a bigger stack only for compilation. I have a suspicion that that change will affect runtime as well and might cause memory problems on production.

Reply:
using `sbt` to run any operation will usually require recompilation, especially if you're not in the sbt console already, I tried those commands and they break without the jvm options

Reply:
I'm not convinced on the other hand that it will produce problems during execution of a pre-packaged jar.
If the stack requirement are caused by shapeless compile-time, as it seems, I don't see any reason why they should affect the runtime

Reply:
I've checked it with `VisualVM` and it looks that when you run ``sbt runConseil`` sbt actually uses the same instance of JVM to compile code and to run an app. However, I've realized that this section is about running an app locally and on production we recommend running it using `java` command. That's good.

Reply:
I lately mentioned to @piotrkosecki the fact that no fork is actually done by sbt in my local dev env, which is weird. I guess we got something amiss, but as you say, the good part is that we advice to run with `java`